app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -222,7 +222,7 @@ private void showBrowserScreen(String url) {
         if (intent.getBooleanExtra(EXTRA_TEXT_SELECTION, false)) {
             TelemetryWrapper.textSelectionIntentEvent();
         } else if (BrowsingSession.getInstance().isCustomTab()) {
-            TelemetryWrapper.customTabIntentEvent();
+            // Telemetry is sent while parsing the custom tab config
```

Comment:
I've rewritten the telemetry to be sent separately (we still need to parse unsupported features during parseCustomTabIntent, but they're store din a list until whenever we actually send telemetry.) Rereview requested for the last 2 patches (one adds telemetry for supported features, one adds the unsupported features) : )!